examples/multi_platform/MixedLib/BUILD

```diff
@@ -3,16 +3,41 @@ load(
     "swift_library",
 )
 load("//apple:apple.bzl", "experimental_mixed_language_library")
+load("//apple:header_map.bzl", "header_map")
 load("//apple:ios.bzl", "ios_unit_test")
 
+filegroup(
+    name = "MixedAnswer.public_headers.h",
+    srcs = [
+        "MixedAnswer.h",
+    ],
+)
+
 experimental_mixed_language_library(
     name = "MixedAnswer",
     srcs = [
         "MixedAnswer.m",
         "MixedAnswer.swift",
     ],
-    hdrs = ["MixedAnswer.h"],
+    hdrs = [
+        ":MixedAnswer.public_headers.h",
+    ],
     enable_modules = True,
+    includes = [
+        "MixedAnswerHeaderMap.hmap",
+    ],
+    objc_copts = ["-I."],
```

Comment:
Can you add a comment about why this is added here or perhaps reference an issue for `experimental_mixed_language_library` to internally manage this when a header map is created?

Reply:
Yep will do, i made: https://github.com/bazelbuild/rules_apple/issues/2371